.gitignore

```diff
@@ -22,4 +22,5 @@ README_PARAMETERS.txt
 *so
 build/
 cython_cygno.c
-reco_run*.txt
\ No newline at end of file
+reco*.txt
```

Comment:
If you use the option to change the output file name, such as `-o reco_tests` , the outputs are:

- reco_tests_run13922_3D.root
- reco_tests_run13922_3D.txt

In this case, the txt is no longer ignored by git. Although looking back the best option would be write something like `*run*.txt` since the "run" is always present.
This is a preference change, I can revert it if you prefer.